packages/cli/src/commands/versions/bump.ts

```diff
@@ -372,51 +372,64 @@ export function createVersionFinder(options: {
     releaseManifest?.packages.map(p => [p.name, p.version]),
   );
   return async function findTargetVersion(name: string) {
-    const existing = found.get(name);
+    let _name = name;
+    const existing = found.get(_name);
     if (existing) {
       return existing;
     }
 
-    console.log(`Checking for updates of ${name}`);
-    const manifestVersion = releasePackages.get(name);
+    console.log(`Checking for updates of ${_name}`);
+
+    const manifestVersion = releasePackages.get(_name);
```

Comment:
This is the most common case for `@backstage` packages, so we'll won't get to the `moved` check